File: 3desafio-dio.edit/des.01/jornadaDoHeroi.js
//Programa para calcular a posição do heroi em um jogo

function jornadaHeroi(posicaoInicial, totalPassos) {
    // Calculando a posição final
    const posicaoFinal = posicaoInicial + totalPassos;

    // Retornando a posição final do herói
    return `A posição final do herói após dar ${totalPassos} passos é: ${posicaoFinal}`;
}

// Exemplo de uso da função
const posicaoInicial = 10;
const totalPassos = 15;

const resultadoJornada = jornadaHeroi(posicaoInicial, totalPassos);
console.log(resultadoJornada);

File: 3desafio-dio.edit/des.01/JornadaDoHeroi.c

int main() {
    int posicaoInicial, totalPassos;

    // Entrada da posição inicial e do número total de passos
    printf("Digite a posição inicial do herói na floresta: ");
    scanf("%d", &posicaoInicial);

    printf("Digite o número total de passos que o herói deve dar: ");
    scanf("%d", &totalPassos);

    // Calculando a posição final
    int posicaoFinal = posicaoInicial + totalPassos;

    // Saída da posição final do herói
    printf("A posição final do herói após dar %d passos é: %d\n", totalPassos, posicaoFinal);

    return 0;
}
